### PR TITLE
Rbac generate

### PR DESCRIPTION
**Pull request by Weeblin**
Made a string of RBAC from GVRs.
#250

### CODE REVIEW

**Comment by tfadeyi** [comment]
Lgtm :+1:  could you add a description to the PR

**Review by tfadeyi — APPROVED** [review]

File: pkg/agent/config.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ type Config struct {
 	OrganizationID string `yaml:"organization_id"`
 	// ClusterID is the cluster that the agent is scanning.
 	ClusterID     string         `yaml:"cluster_id"`
-	DataGatherers []dataGatherer `yaml:"data-gatherers"`
+	DataGatherers []DataGatherer `yaml:"data-gatherers"`
```
Comment (jakexks) [inline pkg/agent/config.go:33]: data-gatherers seems inconsistent with cluster_id and organization_id. Would _ be better?
```suggestion
	DataGatherers []DataGatherer `yaml:"data_gatherers"`
```

Comment (tfadeyi) [reply pkg/agent/config.go:33]: :+1: We'll see if we can change it in another PR. Changing that now would require a couple of other changes in the platform side also.

File: pkg/agent/config.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ type Endpoint struct {
 	Path     string `yaml:"path"`
 }
 
-type dataGatherer struct {
+type DataGatherer struct {
 	Kind     string
 	Name     string
 	DataPath string
```
Comment (jakexks) [inline pkg/agent/config.go:49]: If this field is now exported I recommend adding `yaml:""` tags
```suggestion
type DataGatherer struct {
	Kind     string `yaml:"kind"`
	Name     string `yaml:"name"`
	DataPath string `yaml:"data_path"`
```

**Review by jakexks — APPROVED** [review]

**Pull request merged**